Assets/MRTK/Providers/LeapMotion/Serialized/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyVersion("2.5.0.0")]
```

Comment:
All these added AssemblyInfo.cs files is a good reminder that we should try to revive #7986 / https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8005